stoobly_agent/test/test_helper.py

```diff
@@ -16,6 +16,8 @@
 def reset():
   os.environ[ENV] = TEST
 
+  DataDir._instance = None
```

Comment:
Is this needed? We shouldn't be accessing this directly

Reply:
It's needed to reset the DataDir singleton so recompute the paths